NaCl.py

```diff
@@ -582,6 +583,11 @@ def visitInitializer(self, ctx):
 	def visitFunction(self, ctx):
 		self.nacl_state.save_element(BASE_TYPE_FUNCTION, ctx)
 
+class NaClErrorListener(ErrorListener):
+	def syntaxError(self, recognizer, offendingSymbol, line, column, msg, e):
+		# stop execution on syntax error
+		sys.exit()
```

Comment:
Could we return a non-success exit code here, or would that mess up things in downstream processing?

Reply:
Yeah sure I don't think `sys.exit(1)` will affect anything so I'll update it.
(Note: The actual error is sent to the console/stderr before hitting this so the output will f.ex. be `line 3:1 missing '}' at 'address'` and then the program will exit)